src/Console/Commands/stubs/auth/statamic_auth_tables.php.stub

```diff
@@ -13,21 +13,21 @@ class StatamicAuthTables extends Migration
      */
     public function up()
     {
-        Schema::table('users', function (Blueprint $table) {
+        Schema::table('USERS_TABLE', function (Blueprint $table) {
```

Comment:
This isn't quite working.

If you've customized the `users` table name, then this will fail because it tries to modify an existing table named whatever you've chosen.

I think what should happen is that we detect you're not calling it `users`, we should do a `Schema::create()` block here with everything from [the default users migration](https://github.com/statamic/statamic/blob/master/database/migrations/2014_10_12_000000_create_users_table.php) plus the extra fields here (super, avatar, etc).

If you've called it `users`, we continue to just modify the users table.

Reply:
Hmm, so you basically want to do a check in the `AuthMigration` when dealing with the stubs, so that if a `users` table exist, just add the tables to that existing column. And then if not, it should just create the table instead?

Reply:
Ah gotcha, that makes sense. Where do you think would be a good idea to put the user schema stubs in that case then? 

Just straight in the `AuthMigration` command, inside the `str_replace` or do you think it could work inside the existing migrations folder?

Reply:
You could have 2 additional files in the same directory that hold the `Schema::create()` and `Schema::table()` closures. Read the appropriate file. Maybe? Something like this...

```
public function up()
{
  REPLACEME // either the schema::create() or schema::table() block depending on what they configured

  // the rest of the stuff can stay as-is
  Schema::create('ROLE_USER_TABLE', ... all the stuff 
}
```

```
$contents = str_replace('REPLACEME', $schemaCreateOrTable, $contents);
```

Reply:
I wouldn't bother with the Schema::hasTable check. Just check that they've configured the users table to be called users or not.

Reply:
Ah okay, updated the PR!